src/java.desktop/share/native/libfontmanager/HBShaper_Panama.c:27:10: fatal error: hb.h: No such file or directory

--- jdk23u-jdk-23-ga/make/modules/java.desktop/lib/ClientLibraries.gmk~	2024-08-20 17:57:04.000000000 +0200
+++ jdk23u-jdk-23-ga/make/modules/java.desktop/lib/ClientLibraries.gmk	2024-09-20 10:00:52.460817053 +0200
@@ -279,6 +279,7 @@ endif
 ################################################################################

 ifeq ($(USE_EXTERNAL_HARFBUZZ), true)
+   LIBFONTMANAGER_CFLAGS += $(HARFBUZZ_CFLAGS)
    LIBFONTMANAGER_EXTRA_SRC =
    LIBFONTMANAGER_LIBS += $(HARFBUZZ_LIBS)
 else
